client/src/App.css

```diff
@@ -0,0 +1,193 @@
+* {
```

Comment:
Consider going back and converting this to something more exotic like SCSS or styled components